packages/forms/src/UIForm/fieldsets/Array/ArrayItem.component.js

```diff
@@ -136,7 +136,7 @@ if (process.env.NODE_ENV !== 'production') {
 		index: PropTypes.number.isRequired,
 		isClosed: PropTypes.bool,
 		onRemove: PropTypes.func.isRequired,
-		onReorder: PropTypes.func.isRequired,
+		onReorder: PropTypes.oneOfType([PropTypes.func, PropTypes.bool]).isRequired,
```

Comment:
In some case, boolean can be passed as a callback?
Because on line 22 and 34, we call `onReorder()`
https://github.com/Talend/ui/blob/af7de6bf661e06b7195d5059d6fac74725aa6719/packages/forms/src/UIForm/fieldsets/Array/ArrayItem.component.js#L22 
https://github.com/Talend/ui/blob/af7de6bf661e06b7195d5059d6fac74725aa6719/packages/forms/src/UIForm/fieldsets/Array/ArrayItem.component.js#L34

Reply:
right, should fix this in another way